tests/tracer/test_filters.py

```diff
@@ -1,43 +1,8 @@
-from unittest import TestCase
```

Comment:
Could we keep these tests but just change the import path? Or make a private method these use for now? I'm worried about stripping out tests before something is actually removed.

Reply:
I'll revert this. We don't need to change this test at all. It will just log a deprecation warning in ci.